### PR TITLE
docs: remove placeholder description

### PR DESCRIPTION
**Pull request by bmorelli25**
Every time we update the integration documentation (docs.elastic.co) we have to manually update and work around this description. Can we please remove the WIP language?

### CODE REVIEW

File: packages/cloud_security_posture/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.2.10"
+  changes:
+    - description: Remove placeholder text
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/5163
```
Comment (bmorelli25) [inline packages/cloud_security_posture/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/5187
```

**Comment by kfirpeled** [comment]
Hi @bmorelli25, we weren't aware of that process.
We will fix that asap.

I have a question though, is there a way to commit changes in a way they won't be released? so such processes like the docs generation won't be executed as well?

**Comment by bmorelli25** [comment]
> I have a question though, is there a way to commit changes in a way they won't be released? so such processes like the docs generation won't be executed as well?

Unfortunately, no. Our doc update script was written to work with package-storage v1—so only packages in the production endpoint were published to the public documentation. With the update to package-storage v2, everything exists in the same endpoint and we have no way of selectively publishing documentation.

**Comment by kfirpeled** [comment]
> Unfortunately, no. Our doc update script was written to work with package-storage v1—so only packages in the production endpoint were published to the public documentation. With the update to package-storage v2, everything exists in the same endpoint and we have no way of selectively publishing documentation.

Even if the condition of these updates is set to a version that hasn't been released yet? like 8.7

**Comment by bmorelli25** [comment]
> Even if the condition of these updates is set to a version that hasn't been released yet? like 8.7

That's correct; the script has no understanding of versioning.

**Comment by bmorelli25** [comment]
Talked with @kfirpeled offline. I'm going to close this PR as the text will be updated this week.

**Comment by bmorelli25** [comment]
@kfirpeled, this WIP description continues to disrupt our documentation publishing process. Let me know if this new description is okay with you.

**Comment by bmorelli25** [comment]
@elastic/kibana-cloud-security-posture please can anyone review this one line change

**Pull request closed without merging**